### PR TITLE
loss functions

### PR DESCRIPTION
**Pull request by BarrettLi**
大佬！我又翻译完了。。。

### CODE REVIEW

**Comment by wangyangting** [comment]
貌似提错地方了, 应该是 [apachecn/pytorch-doc-zh](https://github.com/apachecn/pytorch-doc-zh)  ←_←
